Source/WebKitLegacy/Storage/WebStorageNamespaceProvider.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "StorageNamespaceImpl.h"
```

Comment:
What requires a header include instead of a simple forward declaration? It is not directly obvious to me looking at the new code.